risc0/core/Cargo.toml

```diff
@@ -9,7 +9,7 @@ repository = { workspace = true }
 
 [dependencies]
 bytemuck = { version = "1.12", features = ["derive"] }
-bytemuck_derive = "1.9.3"
+bytemuck_derive = "1.8.1" # DO NOT MERGE
```

Comment:
Curious about this one.

Reply:
I'm fine with this on paper. It will usually work out, but in the past we've run into situations where a specific version won't compile for one reason or another (usually conflicts with other dependencies), and so to workaround we've had to pin to a specific version. This usually isn't permanent and a newer version usually does come out so we don't need to do a pin. 

In this specific case I think using `bytemuck_derive = "1"` is appropriate.